app/editor/src/features/admin/reports/ReportFormInstances.tsx

```diff
@@ -108,15 +120,28 @@ export const ReportFormInstance: React.FC = () => {
       />
       <Modal
         headerText="Confirm Resend"
-        body="Are you sure you wish to resend this report instance?"
+        component={
+          <div>
+            <p>Are you sure you wish to resend this report to subscribers?</p>
+            <p>
+              Leave unchecked if you only want to send to subscribers who have not received it yet.
+            </p>
+            <Checkbox
```

Comment:
A failure will allow us to resend only to subscribers who have not received the report.